music_assistant/server/controllers/media/playlists.py

```diff
@@ -319,25 +319,26 @@ async def _get_provider_dynamic_tracks(
         ]
         limit = min(limit, len(playlist_tracks))
         # use set to prevent duplicates
-        final_items = set()
```

Comment:
What's the reasoning to change from set to list ?
I have a habit of preferring sets and tuples wherever I can as its more resource friendlier